src/Compiler.php

```diff
@@ -102,7 +102,7 @@ protected function compute(Registry $registry, Entity $entity): void
             Schema::REPOSITORY   => $entity->getRepository() ?? $this->defaults[Schema::REPOSITORY],
             Schema::CONSTRAIN    => $entity->getConstrain() ?? $this->defaults[Schema::CONSTRAIN],
             Schema::SCHEMA       => $entity->getSchema(),
-            Schema::PRIMARY_KEY  => $this->getPrimary($entity),
+            Schema::PRIMARY_KEY  => $entity->getPrimaryFields()->getNames(),
```

Comment:
Can be incompatible with ORM v1. If PK contains only one field then it should by string, not array.
Same means for FK

Reply:
Changed base from `master` to `2.0`